manager.py

```diff
@@ -79,28 +80,103 @@ def stop(self):
         self._rest_manager.remove_web_handler(self._handler)
         super().stop()
 
-    def get_blocks_structure(self):
-        """ Return file structure under 'blocks' resource """
-        blocks = []
+    def get_blocks_structure(self, branches = 0):
```

Comment:
I can convert it to boolean, was just keeping it consisted with what was coming on from the query parameters. 1/0 is much better in a query parameter than a literal string of "true" or "false".